hack/load-tests/log-fluentbit-test-setup.yaml

```diff
@@ -76,6 +76,18 @@ spec:
       labels:
         app: logs-load-generator
     spec:
+      affinity:
```

Comment:
I'm wondering if we should better use a DaemonSet or at least make the number of replicas dividable by the node count.

Reply:
most load tests require more load generator pods, we have a 2 node setup for all test scenarios I think deployment will evenly distribute pods on two nodes